index.html

```diff
@@ -46,6 +47,190 @@ <h1>I’m Kaweesi Glad to see you!</h1>
+          </ul>
+
+          <div class="hidden1">
+            <button>See Project</button>
+          </div>
+        </section>
+        <section class="tonic">
+          <div><img src="./images/snap1.png" alt="snap" id="snap1" /></div>
+          <div>
+            <h2 id="stories-header1">Tonic</h2>
+          </div>
+
+          <div>
+            <ul id="profs2">
+              <li id="cannopy2">CANOPY</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+
+          <div>
+            <p>
+              A daily selection of privately personalized: no accounts or
+              sign-ups required
+            </p>
+          </div>
+          <div>
+            <ul class="button">
+              <li><button>html</button></li>
+              <li><button>css</button></li>
+              <li><button>javascript</button></li>
+            </ul>
+          </div>
+
+          <div class="hidden2"><button>See Project</button></div>
+        </section>
+        <section class="tonic">
+          <img src="./images/snap3.png" alt="snap" />
+          <h2 id="stories-header2">Multi-post stories</h2>
+
+          <ul id="profs3">
+            <li id="cannopy3">CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+
+          <p>
+            A daily selection of privately personalized: no accounts or sign-ups
+            required
+          </p>
+          <ul class="button">
+            <li><button>html</button></li>
+            <li><button>css</button></li>
+            <li><button>javascript</button></li>
+          </ul>
+          <div class="hidden3">
+            <button>See Project</button>
+          </div>
+        </section>
+      </div>
+      <div class="abt">
+        <section class="about">
+          <div>
+            <h2>About</h2>
+            <h2>Myself</h2>
+          </div>
+
+          <p>
+            Hello I'm a software developer! I can help you build your
+            project,feature or website.Look through some of my work and
+            experience! If you like what you see and have the project you need
+            coded,don't hesitate to contact me.
+          </p>
+
+          <a id="connect1" href="index.html">LET'S CONNECT</a>
+          <ul class="content-list">
+            <li>
+              <a href=""> <img src="images/twitter.png" alt="Twitter" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="images/Linkedin.png" alt="LinkedIn" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="./images/Vector-1.png" alt="image" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="./images/vector-2.png" alt="image" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="images/medium.png" alt="Media" /></a>
+            </li>
+          </ul>
+
+          <div class="hidden4">
+            <button>See Project</button>
```

Comment:
Kindly change the button text to say `Get my Resume` instead of `See Project` as in the Figma Design 😃

![image](https://user-images.githubusercontent.com/73714615/185751881-d06af874-896e-4daf-b2f3-0ca133a0a67c.png)